docs/how-to/ocm-control-plane-walkthrough.md

```diff
@@ -6,7 +6,10 @@ You will also deploy a simple application that uses that gateway for ingress and
 We will start with a single cluster and move to multiple clusters to illustrate how a single gateway definition can be used across multiple clusters and highlight the automatic TLS integration and also the automatic DNS load balancing between gateway instances.
 
 ## Requirements
-- [Kind](https://kind.sigs.k8s.io/)
+- [kind](https://kind.sigs.k8s.io/)
+- [operator-sdk](https://sdk.operatorframework.io/docs/installation/)
```

Comment:
Is operator-sdk still needed?

Reply:
Thought for another day, but we really don't have to deploy the kuadrant operator via OLM or deploy OLM with the operator-sdk. I guess for now, since that's what we do we will have to leave it here, but it would be better if we didn't have to.

Reply:
While it would be nice to address this here as we did with istio it seems a bigger change so perhaps we could create a follow on issue for it? We plan to create a follow on issue for having a Dockerfile manage the dependencies so this might not be so bit a problem after that?

Reply:
Yes, I'd leave it for now. If we can get a docker image working it doesn't really matter.